archlinuxcn/librime-sbxlm-git/PKGBUILD

```diff
@@ -13,7 +13,7 @@ epoch=2
 pkgdesc="Rime input method engine (声笔系列码 GIT version)"
 arch=('x86_64')
 url="https://github.com/sbxlmdsl/librime"
-license=('GPL3')
+license=('BSD-3-Clause')
```

Comment:
The BSD-3-Clause license is a type of open source software license that is not part of the [standard licenses](https://gitlab.archlinux.org/archlinux/packaging/packages/licenses/-/blob/main/PKGBUILD?ref_type=heads#L40). If you use software under this license, you need to copy the original license file to the /usr/share/licenses/pkgname directory.